grammar.js

```diff
@@ -456,8 +460,16 @@ module.exports = grammar({
       optional(',')
     ),
 
+    star_expression: $ => seq(
+      '*',
+      $._expression,
+    ),
```

Comment:
There's already something called `list_splat` (which represents unpacking also). Should that be used here instead of adding `star_expression`?